cura/CuraActions.py

```diff
@@ -181,5 +187,63 @@ def setBuildPlateForSelection(self, build_plate_nr: int) -> None:
 
         Selection.clear()
 
+    @pyqtSlot()
+    def cut(self) -> None:
+        self.copy()
+        self.deleteSelection()
+
+    @pyqtSlot()
+    def copy(self) -> None:
+        # Convert all selected objects to a Savitar scene
+        savitar_scene = Savitar.Scene()
+        mesh_writer = cura.CuraApplication.CuraApplication.getInstance().getMeshFileHandler().getWriter("3MFWriter")
+        for scene_node in Selection.getAllSelectedObjects():
+            savitar_node = mesh_writer._convertUMNodeToSavitarNode(scene_node)
```

Comment:
I don't think that this is a good idea. This is a protected function.

Reply:
I think it would be better to move the entire savitar logic to the 3mf writer. So basicly we would need to add a "toSceneString" (or something like that) function that accepts a list of nodes to the writer

Reply:
https://github.com/Ultimaker/Cura/pull/16383/commits/f8b3fb3d67890b5ee3d2aeac945c9ff136cf62bf